Src/Support/GoogleApis/Apis/[Media]/Upload/ResumableUpload.cs

```diff
@@ -522,15 +643,16 @@ protected virtual void ProcessResponse(HttpResponseMessage httpResponse)
                 }.CreateRequest();
 
             // Prepare next chunk to send.
-            if (StreamLength != UnknownSize)
+            int contentLength;

```

Comment:
The reason I didn't is that the Prepare...() methods are impure, and I generally prefer to only use the ?: operator when calling pure methods.
I accept this is probably a personal preference, I'll certainly change it if you like ;)